bundles/org.openhab.binding.insteon/src/main/resources/device_types.xml

```diff
@@ -343,6 +345,18 @@ Example entry:
      	<feature name="keypadbuttonc">KeyPadButton5</feature>
      	<feature name="keypadbuttond">KeyPadButton6</feature>
      </feature_group>
+     <feature_group name="fastonoff_button_group" type="FastOnOffButtonGroup">
+        <feature name="fastonoffbuttona">FastOnOffButton3</feature>
```

Comment:
This product's description mentions 6 buttons but you only use 4?

Reply:
The two other buttons are load on/off buttons

Reply:
Yes, as @robnielsen  said, Load on/off are two of the 6 buttons with the remaining 4 being scene buttons.   On the 8 button variant, ON/Off becomes a toggling single button and 7 more scene buttons.